interop-cats/shared/src/main/scala/zio/interop/cats.scala

```diff
@@ -46,7 +46,8 @@ abstract class CatsEffectPlatform
   }
 
   object implicits {
-    implicit final def ioTimer[E]: effect.Timer[IO[E, *]] = ioTimer0.asInstanceOf[effect.Timer[IO[E, *]]]
+    final def ioTimer[E]: effect.Timer[IO[E, *]]                  = ioTimer0.asInstanceOf[effect.Timer[IO[E, *]]] // bincompat?
+    implicit final def rioTimer[R, E]: effect.Timer[ZIO[R, E, *]] = ioTimer0.asInstanceOf[effect.Timer[ZIO[R, E, *]]]
 
     private[this] val ioTimer0: effect.Timer[IO[Any, *]] =
```

Comment:
Might change this signature to 

```suggestion
    private[this] val ioTimer0: effect.Timer[ZIO[Any, Nothing, *]] =
```

To make it more understandable that its type-safe to cast, since `ZIO[Any, Nothing, ?] <: ZIO[R, E, ?] ∀ R, E`